client/app/components/dynamic-form/fields/CheckboxField.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import Checkbox from "antd/lib/checkbox";
-import getFieldLabel from "../getFieldLabel";
+import getFieldLabel from '@/components/dynamic-form/getFieldLabel';
```

Comment:
Basically, in this case I'd prefer using relative import. These components are in fact a part of a single, bigger one, and those smaller ones shouldn't (and aren't) used separately. @elado WDYT?

Reply:
There are a lot of cases relative imports look better IMO. For Eslint, though, is either yay yay or no no. The idea behind such linter rules is enforce a tree-like file structure.

I pushed the changes so that I got your opinion on this. Maybe we can remove it. The main advantage here is autofix that is good for most cases.

Reply:
Also prefer relative imports when they are not crazy `../../../`.

Reply:
The reason is that it's easier to move components around without breaking imports. True - VSCode has update imports feature, but that wouldn't be necessary with relative ones.
Nested files don't need to know where they are in the tree, they behave like a single unit together with other files in the same folder.

Reply:
Can we just disable the relative imports bit, but keep the other linting rules `import` provides? It sounds like we don't have consensus, here, so we shouldn't start enforcement, IMHO.

Reply:
That's just what I did. There are actually two plugins in play here: prefer aliases is the one we want to hold off.